### ISSUE TITLE
[Quest][Outland] The Earthbinder in Hellfire Peninsula

### ISSUE THREAD

**Issue by Amplefere** [dalaranwow/dalaran-wow#4888]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behavior**: Tell us what is happening
Quest: "The Earthbinder" started from Tola'thion in Hellfire Peninsula unable to be turned in to Earthbinder Galandria Nightbreeze at location 16.0, 51.6, no "?" over NPC, unable to interact

**Expected behaviour**: Tell us what should happen instead
Should be able to interact with NPC to turn in quest

**Steps to reproduce the problem**:

1. Start quest
2. "Try" to complete
3. 

**Include proofs for this behaviour**
http://wowwiki.wikia.com/wiki/Quest:The_Earthbinder

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @Amplefere and thank you for the report.
 Issue is related to #2822. 
Npc  http://www.wowhead.com/npc=19294/earthbinder-galandria-nightbreeze#abilities is famous of getting stucked due to the mini event she does after completing quest http://www.wowhead.com/quest=10349/the-earthbinder. Think I found the reason why she could get stucked so this is the improved SAI for her, let's hope it won't be possible to this to happen again.

``` sql
-- Earthbinder Galandria Nightbreeze SAI
SET @ENTRY := 19294;
UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`=@ENTRY;
DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
(@ENTRY,0,0,0,38,0,100,0,1,1,0,0,1,0,0,0,0,0,0,21,100,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Data Set 1 1 - Say Line 0"),
(@ENTRY,0,1,2,20,0,100,512,10349,0,0,0,80,@ENTRY*100+00,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Quest '' Finished - Run Script"),
(@ENTRY,0,2,5,61,0,100,512,10349,0,0,0,48,1,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Quest '' Finished - Set Active On"),
(@ENTRY,0,3,0,38,0,100,512,2,2,0,0,80,@ENTRY*100+01,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Data Set 2 2 - Run Script"),
(@ENTRY,0,4,7,11,0,100,512,0,0,0,0,81,3,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Respawn - Set Npc Flags Gossip & Questgiver"),
(@ENTRY,0,5,0,61,0,100,512,10349,0,0,0,59,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Quest '' Finished - Set Run Off"),
(@ENTRY,0,6,0,6,0,100,0,0,0,0,0,45,3,0,0,0,0,0,19,20599,100,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Just Died - Set Data 3 0"),
(@ENTRY,0,7,0,61,0,100,0,0,0,0,0,59,1,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Respawn - Set Run On"),
(@ENTRY,0,8,0,0,0,100,0,0,0,10000,14000,11,33844,0,0,0,0,0,2,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - In Combat - Cast 'Entangling Roots'"),
(@ENTRY,0,9,0,0,0,100,0,1600,1600,13000,15000,11,31401,0,0,0,0,0,2,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - In Combat - Cast 'Moonfire'"),
(@ENTRY,0,10,0,0,0,100,0,1700,1700,0,0,11,21807,64,0,0,0,0,2,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - In Combat - Cast 'Wrath'"),
(@ENTRY,0,11,0,21,0,100,0,0,0,0,0,81,3,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Reached Home - Set Npc Flags Gossip & Questgiver");

-- Actionlist SAI
SET @ENTRY := 1929400;
DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=9;
INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
(@ENTRY,9,0,0,0,0,100,0,0,0,0,0,81,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Script - Set Npc Flag "),
(@ENTRY,9,1,0,0,0,100,0,0,0,0,0,66,0,0,0,0,0,0,21,50,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Script - Set Orientation Closest Player"),
(@ENTRY,9,2,0,0,0,100,0,0,0,0,0,1,1,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Script - Say Line 1"),
(@ENTRY,9,3,0,0,0,100,0,6000,6000,0,0,69,0,0,0,0,0,0,8,0,0,0,-286.767,4729.43,18.4418,1.72788,"Earthbinder Galandria Nightbreeze - On Script - Move To Position"),
(@ENTRY,9,4,0,0,0,100,0,11000,11000,0,0,5,16,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Script - Play Emote 16"),
(@ENTRY,9,5,0,0,0,100,0,3000,3000,0,0,5,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Script - Play Emote 0"),
(@ENTRY,9,6,0,0,0,100,0,0,0,0,0,50,184450,60,0,0,0,0,8,0,0,0,-287.019,4731.63,18.217,2.58308,"Earthbinder Galandria Nightbreeze - On Script - Summon Gameobject 'Crimson Crystal Shard'"),
(@ENTRY,9,7,0,0,0,100,0,4000,4000,0,0,1,2,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Script - Say Line 2"),
(@ENTRY,9,8,0,0,0,100,0,0,0,0,0,12,20599,3,60000,1,0,0,8,0,0,0,-288.19,4733.63,18.2982,5.044,"Earthbinder Galandria Nightbreeze - On Script - Summon Creature 'Lured Colossus'");

-- Actionlist SAI
SET @ENTRY := 1929401;
DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=9;
INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
(@ENTRY,9,0,0,0,0,100,0,0,0,0,0,69,0,0,0,0,0,0,8,0,0,0,-294.766,4715.08,28.1862,0.20944,"Earthbinder Galandria Nightbreeze - On Script - Move To Position"),
(@ENTRY,9,1,0,0,0,100,0,13000,13000,0,0,66,0,0,0,0,0,0,1,0,0,0,0,0,0,0.20944,"Earthbinder Galandria Nightbreeze - On Script - Set Orientation Home Position"),
(@ENTRY,9,2,0,0,0,100,0,0,0,0,0,81,3,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Script - Set Npc Flags Gossip & Questgiver"),
(@ENTRY,9,3,0,0,0,100,0,0,0,0,0,48,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Earthbinder Galandria Nightbreeze - On Script - Set Active Off");


-- Lured Colossus SAI
SET @ENTRY := 20599;
UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`=@ENTRY;
DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
(@ENTRY,0,0,0,6,0,100,512,0,0,0,0,45,2,2,0,0,0,0,19,19294,50,0,0,0,0,0,"Lured Colossus - On Just Died - Set Data 2 2"),
(@ENTRY,0,1,0,54,0,100,0,0,0,0,0,48,1,0,0,0,0,0,1,0,0,0,0,0,0,0,"Lured Colossus - On Just Summoned - Set Active On"),
(@ENTRY,0,2,0,38,0,100,0,3,0,0,0,41,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Lured Colossus - On Data Set 3 0 - Despawn Instant");
```

**Comment by Amplefere** [comment]
VERIFIED: Was able to successfully turn in quest and continue quest chain.

Thanks!

**Comment by Rushor** [comment]
DB/Quest: The Earthbinder - Prevent creature from stucking in one place

* thanks to Teridan
* closes: https://github.com/dalaranwow/dalaran-wow/issues/4888

The issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
